SpatialGDK/Source/SpatialGDK/Private/Utils/RPCContainer.cpp

```diff
@@ -66,8 +193,36 @@ bool FRPCContainer::ObjectHasRPCsQueuedOfType(const Worker_EntityId& EntityId, E
 
 	return false;
 }
+ 
+void FRPCContainer::BindProcessingFunction(const FProcessRPCDelegate& Function)
+{
+	ProcessingFunction = Function;
+}
 
-bool FRPCContainer::ApplyFunction(const FProcessRPCDelegate& FunctionToApply, const FPendingRPCParams& Params)
+bool FRPCContainer::ApplyFunction(FPendingRPCParams& Params)
 {
-	return FunctionToApply.Execute(Params);
+	FRPCErrorInfo ErrorInfo;
+
+	if (ProcessingFunction.IsBound())
+	{
+		ErrorInfo = ProcessingFunction.Execute(Params);
+	}
+	else
+	{
+		ErrorInfo = FRPCErrorInfo{ nullptr, nullptr, true, ERPCQueueType::Unknown, ERPCResult::NoProcessingFunctionBound };
```

Comment:
This looks like a different kind of error that we shouldn't normally get, so maybe worth doing a warning/error that isn't an RPC error?

Reply:
Addressed.